src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -1017,23 +1065,34 @@ export class StoreQuickstartSetup extends SfdxCommand {
     private async updateFlowAssociatedToCheckout(): Promise<void> {
         if (await this.statusFileManager.getValue('updatedFlowAssociatedToCheckout')) return;
         this.ux.log('Updating flow associated to checkout.');
-        const checkoutMetaFolder = `${this.storeDir}/experience-bundle-package/unpackaged/experiences/${
-            this.varargs['communityExperienceBundleName'] as string
-        }/views/`;
-        // Do a case insensitive grep and capture file
-        const greppedFile = fs.readdirSync(checkoutMetaFolder).filter((f) => f.toLowerCase() === 'checkout.json')[0];
-        // This determines the name of the main flow as it will always be the only flow to terminate in "Checkout.flow"
-        const flowDir = this.storeDir + '/../force-app/main/default/flows';
-        const mainFlowName = path
-            .basename(fs.readdirSync(flowDir).filter((f) => f.endsWith('Checkout.flow-meta.xml'))[0])
-            .split('.')[0];
-        fs.writeFileSync(
-            checkoutMetaFolder + '/' + greppedFile,
-            fs
-                .readFileSync(checkoutMetaFolder + '/' + greppedFile)
-                .toString()
-                .replace('sfdc_checkout__CheckoutTemplate', mainFlowName.toString())
-        );
+        if (!this.isUsingDigitalExperienceBundle) {
```

Comment:
What is the use case for this flow? Does the checkout for 'DigitalExperienceBundle' work with out this flow/code

Reply:
The checkout in b2b lwr works fine without it. I couldn't find the equivalent configuration in the DigitalExperienceBundle folder which may mean this flow name override is not necessary.

Reply:
What happens if you try to remove this for aura store? I feel we should understand why exactly this piece of code is NOT required and add a comment

Reply:
I will try to remove it and see what happens 👍

Reply:
Have you tried removing this code? May be worth checking with the checkout team and understand the flow as a followup story/work

Reply:
We cannot remove this part, because if I do that and create a B2B Aura store, I get this error while trying to checkout:
`The flow failed to start because the provided values were not valid: We ran into a problem trying to run your flow. Make sure your checkout flow is assigned to a store or contact your account rep.`